cloud/terraform/terraform.tfvars

```diff
@@ -0,0 +1,10 @@
+
+frontend_name = "frontend-vm"
```

Comment:
to fie nie duplikuje z varaibles.tf?

Reply:
nie, w variables.tf te zmienne są deklarowane, w tfvars są definiowane wartości. To jest po to żeby były bardziej przejrzyste. tfvars nadpisuje domyślne wartości zmiennych. To jest świadomie zrobione.